pep-0001.txt

```diff
@@ -421,8 +421,10 @@ Each PEP should have the following parts/sections:
 3. Motivation -- The motivation is critical for PEPs that want to
    change the Python language, library, or ecosystem.  It should
    clearly explain why the existing language specification is
-   inadequate to address the problem that the PEP solves.  PEP
-   submissions without sufficient motivation may be rejected outright.
+   inadequate to address the problem that the PEP solves.  This can
+   include collecting documented support for the PEP from important
+   projects in the Python ecosystem.  PEP submissions without
+   sufficient motivation may be rejected outright.
```

Comment:
I would suggest leaving out the "outright", but otherwise LGTM.

Reply:
```suggestion
   sufficient motivation may be rejected.
```